native/components/message-markdown.react.js

```diff
@@ -0,0 +1,79 @@
+// @flow
+
+import { Component } from 'react';
```

Comment:
I prefer `import * as React from 'react';`, and then you can use `React.Component`